pkg/microservice/aslan/core/common/service/render.go

```diff
@@ -48,12 +49,19 @@ type KVPair struct {
 	Value interface{} `json:"value"`
 }
 
+type ValuesDataArgs struct {
+	YamlSource    string      `json:"yamlSource,omitempty"`
+	GitRepoConfig *RepoConfig `json:"gitRepoConfig,omitempty"`
+}
+
 type RenderChartArg struct {
-	EnvName        string    `json:"envName,omitempty"`
-	ServiceName    string    `json:"serviceName,omitempty"`
-	ChartVersion   string    `json:"chartVersion,omitempty"`
-	OverrideValues []*KVPair `json:"overrideValues,omitempty"`
-	OverrideYaml   string    `json:"overrideYaml,omitempty"`
+	EnvName        string                     `json:"envName,omitempty"`
+	ServiceName    string                     `json:"serviceName,omitempty"`
+	ChartVersion   string                     `json:"chartVersion,omitempty"`
+	OverrideValues []*KVPair                  `json:"overrideValues,omitempty"`
+	OverrideYaml   string                     `json:"overrideYaml,omitempty"`
+	ValuesData     *ValuesDataArgs            `json:"valuesData,omitempty"`
+	YamlData       *templatemodels.CustomYaml `json:"yaml_data,omitempty"`
```

Comment:
`json:"yaml_data,omitempty"` -> `json:"yamlData,omitempty"`

Reply:
there are multiple apis return the `yaml_data` property, to make it easy to handle for front-end matess i use the the same json tag. It is wired indeed(also happens in other apis), i suggest we can unify the style of json tag globally to solve those iuuses.